### PR TITLE
fix api_reference

### PR DESCRIPTION
**Pull request by MrLeea-13155bc**

### CODE REVIEW

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/api/deviceshifu-configmap.md [hunk 1]
```diff
@@ -72,7 +71,8 @@ sidebar_position: 3
 ## DeviceShifuTelemetrySettings
 
 `DeviceShifuTelemetrySettings` 是指与 ***deviceShifu*** 监测相关的设置。
-
+- **defaultPushToServer** (bool) ***deviceShifu*** 默认将所有遥测进行推送服务。
```
Comment (tomqin93) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/api/deviceshifu-configmap.md:74]: should mention default values, same for other occurrences

**Review by tomqin93 — COMMENTED** [review]
added some comment

**Comment by Yang-Xijie** [comment]
It will be great to remove `WORK IN PROGRESS` if you have finished these docs.

**Comment by Yang-Xijie** [comment]
It will be great to add the version mark, e.g. `shifu v0.3.0`, at the top of these docs.

**Comment by MrLeea-13155bc** [comment]
> It will be great to add the version mark, e.g. `shifu v0.3.0`, at the top of these docs.

you come to add it , for i dont know where to add it. i try to add some but they are inappropriate

**Comment by Yang-Xijie** [comment]
image

It seems that you didn't modify `references/api/deviceshifu-deployment`. Is this file up-to-date? @MrLeea-13155bc

**Comment by MrLeea-13155bc** [comment]
> image
> 
> 
> 
> It seems that you didn't modify `references/api/deviceshifu-deployment`. Is this file up-to-date? @MrLeea-13155bc 

I don't think this file need to update, for a new feature will not modify deployment file

**Comment by Yang-Xijie** [comment]
I will format these files soon.

**Comment by Yang-Xijie** [comment]
This PR is now ready to merge.

**Pull request merged**